src/index.js

```diff
@@ -9,14 +9,25 @@ import TrayManager from './server/tray_manager'
 import StoryType from './model/story_type'
 import ReadCache from './model/read_cache'
 import Logger from './logger'
+import windowStateKeeper from 'electron-window-state'
 
 var server = new Server()
 
+var mainWindowState = windowStateKeeper('main', {
+  width: 400,
+  height: 400
+}, {
+  folderName: 'Configuration',
```

Comment:
Will the folder be created if it's not there? Seems the API is a bit redundant. Maybe it could just take a path instead and it's up to the caller to create folder etc.